gerrit/chain.go

```diff
@@ -65,20 +65,20 @@ func (s *Serie) CheckIntegrity() error {
 	return nil
 }
 
-// FilterAllChangesets applies a filter function on all of the changesets in the series.
+// AllChangesets applies a filter function on all of the changesets in the chain.
 // returns true if it returns true for all changesets, false otherwise
-func (s *Serie) FilterAllChangesets(f func(c *Changeset) bool) bool {
+func (s *Chain) AllChangesets(f func(c *Changeset) bool) bool {
 	for _, changeset := range s.ChangeSets {
-		if f(changeset) == false {
+		if f(changeset) {
```

Comment:
This inverts the logic and it doesn't seem like there's a change at the call site?

Reply:
Thanks! Seems my keyboard got stuck and I didn't add the `!` when simplifying the expression. Will fix.

Reply:
Fixed.